### PR TITLE
Replace checkout with switch/restore

### PR DESCRIPTION
**Pull request by outsideris**
## Overview
**TL;DR**
Replace `checkout` with `switch` and `restore` because the latest git encourages to use `switch`/`restore` instead `checkout`.

### Next Steps
I can't change PDF files, because I don't know how I can generate them.

### Review
I tried to change them in all supported languages using Google Translate. 
But I'm not sure because I can read only Korean and English.

### CODE REVIEW

**Comment by brianamarie** [comment]
Thank you for this PR, @outsideris! 

Before merging, I'd like to confirm with the @github/ps-implementation-engineers that this change works for them. We may also want to make sure this change is implemented in the [training manual](https://github.com/githubtraining/training-manual) for consistency. 

@github/ps-implementation-engineers, what do you think?

cc @github/ps-programs

**Review by parkerbxyz — APPROVED** [review]
Thank you for these contributions, @outsideris! ✨ 

I can’t vouch for the non-english translations, but the commands looks great!

@brianamarie Can you verify that is it safe to make changes to Git commands in the non-english versions before we merge. I know some repos take care of the translations in a separate process, using the english version to reference what needs to be updated. Not sure if that’s the case here.

**Comment by parkerbxyz** [comment]
> We may also want to make sure this change is implemented in the [training manual](https://github.com/githubtraining/training-manual) for consistency.

This is on our radar. 😎

**Review by Dmt80 — APPROVED** [review]

File: downloads/github-git-cheat-sheet.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Branches are an important part of working with Git. Any commits you make will be
 
 Creates a new branch
 
-```$ git checkout [branch-name]```
+```$ git switch [branch-name]```
```
Comment (rwnfoo) [inline downloads/github-git-cheat-sheet.md:41]:
```suggestion
```$ git switch -c [branch-name]```
```

Creating a new branch requires `-c`

File: downloads/id/github-git-cheat-sheet.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Daftar semua cabang lokal di repositori saat ini
 
 Membuat cabang baru
 
-```$ git checkout [nama-cabang]```
+```$ git switch [nama-cabang]```
```
Comment (rwnfoo) [inline downloads/id/github-git-cheat-sheet.md:96]:
```suggestion
```$ git switch -c [nama-cabang]```
```

macamtu lah kalau nak buat cabang baru

File: downloads/id/github-git-cheat-sheet.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ Daftar semua cabang lokal di repositori saat ini
 
 Membuat cabang baru
```
Comment (rwnfoo) [inline downloads/id/github-git-cheat-sheet.md:94]:
```suggestion
Mewujudkan cabang baru
```

File: downloads/it/github-git-cheat-sheet.md [hunk 4]
```diff
@@ -102,7 +102,7 @@ Elenca tutti i branch nel repository corrente
 Crea un nuovo branch
 
 
-```$ git checkout [branch-name]```
+```$ git switch [branch-name]```
```
Comment (rwnfoo) [inline downloads/it/github-git-cheat-sheet.md:105]:
```suggestion
```$ git switch -c [branch-name]```
```

File: downloads/zh_CN/github-git-cheat-sheet.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ leadingpath: ../../../
 
 创建一个新分支
 
-```$ git checkout [branch-name]```
+```$ git switch [branch-name]```
```
Comment (rwnfoo) [inline downloads/zh_CN/github-git-cheat-sheet.md:41]:
```suggestion
```$ git switch -c [branch-name]```
```

新分支需要加个`- c`

**Comment by rwnfoo** [comment]
Thank you @outsideris for helping with this! 🥳 

There are 2 commands that need to be straightened out, which looks like you're aware of: 
`git switch` for branches that already exist and `git switch -c` to replace `git checkout -b`. However, there is an error in the original docs where the command for creating a new branch is `git checkout` instead of `git checkout -b`, [this](downloads/github-git-cheat-sheet.md) is one example.

I can only read 2 other languages where this do not lined up, so I made suggestions in the Malay and Chinese version. But I think it needs to be corrected for other languages too.

**Comment by outsideris** [comment]
@rwnfoo Good point!! I will change other languages as well even I can read only English and Korean.

**Comment by outsideris** [comment]
I fixed creating a branch with `-c` option. I checked it by translating them on Google Translate.

**Review by Dmt80 — APPROVED** [review]

**Comment by rwnfoo** [comment]
Thank you so much @outsideris for the work! ✨ 

@parkerbxyz - Wondering if we should hold on this PR until `git switch` is no longer an experimental feature.

**Review by lucalves — APPROVED** [review]

**Review by parkerbxyz — APPROVED** [review]
Looks good. Thanks for your contributions, everyone! ✨

**Comment by parkerbxyz** [comment]
> @parkerbxyz - Wondering if we should hold on this PR until `git switch` is no longer an experimental feature.

Let’s proceed here like we did in https://github.com/githubtraining/training-manual/pull/281 and ship this! :shipit:

**Pull request merged**